deploy/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: infinispan-operator
       containers:
         - name: infinispan-operator
-          image: jboss/infinispan-operator:latest
+          image: quay.io/infinispan/operator:2.0.x
```

Comment:
@rigazilla we should also regen install-bundle.yaml

Reply:
yep, done thanks!